packages/vue-app/template/store.js

```diff
@@ -3,73 +3,87 @@ import Vuex from 'vuex'
 
 Vue.use(Vuex)
 
-const files = require.context('@/<%= dir.store %>', true, /^\.\/(?!<%= ignorePrefix %>)[^.]+\.(<%= extensions %>)$/)
-const filenames = files.keys()
-
-// Store
 let storeData = {}
 
-// Check if {dir.store}/index.js exists
-const indexFilename = filenames.find(filename => filename.includes('./index.'))
+let files;
 
-if (indexFilename) {
-  storeData = getModule(indexFilename)
-}
+(function updateModules() {
```

Comment:
@manniL quick tip, it's possible to write:

```js
void function() {

}()
```

Reply:
Thanks, implemented ☺️